### PR TITLE
Terms

### LINKED ISSUE #1269: Add EULA to API
for to clarify the terms under which the API may be used.

### PR DESCRIPTION
**Pull request by BenJam**
Adds a basic set of terms for using the API. Does not include a link from the API page at this point. This PR is not to be merged until reviewed by legal representative and a representative of Brave new Software. 

Fixes #1269

### CODE REVIEW

**Comment by BenJam** [comment]
Picking this back up:

- [ ] add links to the footer
- [ ] expand the terms and conditions to extend to the whole service
- [ ] update privacy policy to echo that of depednedncyci 
- [ ] add an opt in process and record per user on the database
- [ ] add a banner notice for users that have not accepted 
- [ ] make sure on sign up that the terms/privacy notice are linked up for new users

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -24,4 +24,10 @@ def data
     @platforms = Project.popular_platforms(:facet_limit => 40).first(28)
 
   end
+  def terms
+    
+  end
+  def terms
```
Comment (andrew) [inline app/controllers/pages_controller.rb]: You've got this method twice

File: app/views/pages/privacy.html.erb [hunk 2]
```diff
@@ -89,7 +96,7 @@
       From time to time we may modify or update our privacy policy. We will do so by updating this page.
     </p>
     <p>
-      <b>Last update: 20th February 2016</b>
+      <b>Last update: 27th September 2016</b>
```
Comment (M-Zuber) [inline app/views/pages/privacy.html.erb]: Shouldn't the year be changed as well?

File: app/views/shared/_notice.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<% if current_user && current_user.optin == false %>
+    <div class='alert alert-danger'>
+      We have updated our <a href='/terms'>terms of service</a> and <a href='/privacy'>privacy policy</a>. If you do not accept these changes <b>your account will be removed</b> on 31st December 2016. <%= link_to 'Accept', optin_account_path, method: 'put', class: 'btn btn-success btn-sm', rel: 'nofollow', title: 'Accept the new terms of service and privacy policy' %>
```
Comment (M-Zuber) [inline app/views/shared/_notice.html.erb]: Year should probably be changed here as well

File: app/views/pages/privacy.html.erb (hunk 2 above)
Comment (andrew) [reply app/views/pages/privacy.html.erb]: 👍

**Comment by andrew** [comment]
Do we need to add something in here: 
screen shot 2017-09-27 at 1 48 46 pm

**Comment by andrew** [comment]
Updated dropdown too: 
screen shot 2017-09-27 at 2 06 52 pm

**Pull request merged**